pkg/collector/corechecks/net/wlan/wlan_nix.go

```diff
@@ -3,13 +3,13 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-//go:build !darwin
+//go:build freebsd || linux || netbsd || openbsd || solaris || dragonfly || aix
```

Comment:
```suggestion
//go:build !darwin && !windows
```
How about this?

Reply:
yeah, this is better, I just copied above from some other source, thank you